x/registry/keeper/msg_server_who_is.go

```diff
@@ -86,7 +86,7 @@ func (k msgServer) UpdateWhoIs(goCtx context.Context, msg *types.MsgUpdateWhoIs)
 			val.AddAlsoKnownAs(a, false)
 		}
 	}
-	val.Controllers = doc.ControllersAsString()
+	val.Controllers = doc.GetController()
```

Comment:
This method should be named `GetControllers`. I know it's not part of this PR but we should at least put an issue or just change it here.

Reply:
If anything WhoIs should be renamed to `Controller` that is the compliant property with the DID W3 Specification

Reply:
we can make an issue for this one.